guides/m1x/ce18-ee113-home.html

```diff
@@ -32,7 +32,7 @@ <h1>Magento Community Edition (CE) 1.8 and Enterprise Edition (EE) 1.13 Document
 		<ul><li><a href="{{ site.m1xgdeurl }}install/installing.html" target="_blank">Installing and Upgrading to Magento Community Edition (CE) and Magento Enterprise Edition (EE)</a></li>
```

Comment:
Please fix merge conflicts and push back up for review.

Reply:
@shrielenee Resolved conflicts.